guides/common/modules/proc_republishing-repository-metadata.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Republishing_Repository_Metadata_{context}"]
+= Republishing Repository Metadata
+
+Republishing repository metadata is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
+
+Use this procedure with caution.
+In most cases, {Team} recommends a complete repository sync or publishing a new Content View version.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the product that includes the repository for which you want to republish metadata.
+. Navigate to the repository where you want to republish metadata.
+. To republish metadata for the repository, select *Republish Repository Metadata* from the *Select Action* menu.
++
+[NOTE]
+====
+This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source instead of being generated locally.
```

Comment:
```suggestion
This action is not available for repositories that use the *Complete Mirroring* policy because the metadata is copied verbatim from the upstream source of the repository.
```